### DISCUSSION TITLE
Find out Server Version Info

### DISCUSSION THREAD

**Discussion by yodaphone** [dani-garcia/vaultwarden#4877] [General]
Hi

This might be a really dumb question, but how do i find out the server version I'm running?

**Comment by BlackDex** [comment]
Most easy way is to check `/admin/diagnostics`.

Or by running the vaultwarden server via the cli `./vaultwarden --version` or via docker `docker exec -it vaultwarden /vaultwarden --version`

**Reply by yodaphone** [reply]
thanks

**Reply by stefan0xC** [reply]
> Most easy way is to check `/admin/diagnostics`.

Since you need to [enable the `/admin` page](https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page) for that, I'd argue that the most easiest is actually opening `/api/config` as it should work in any case. (The only thing to note would be that you need to look for `.server.version` for the vaultwarden version as the `.version` is hard-coded to `"2024.2.0"` to simulate a Bitwarden server version.)
https://github.com/dani-garcia/vaultwarden/blob/cbdb5657f1bd234a98972df5c11e0492f10f08a8/src/api/core/mod.rs#L201-L207

edit: nvm. this has been removed in #4885

**Reply by MR4online** [reply]
would be great to see the current version in the web ui :)

**Reply by BlackDex** [reply]
Logoutz and check the login page.

**Reply by MR4online** [reply]
@BlackDex no. I'm talking about the Server Version like this thread. Should be something like 1.32.5
The Web UI only prints the 'Vaultwarden Web Version 2024.5.1'

**Reply by BlackDex** [reply]
We can't. This information is not able to be viewed/fetched in the web-vault code without make a lot of changes.

Just use the `/admin/diagnostic`.
Also, a lot of people find it insecure to show this at all.
